t/unit/utils/test_json.py

```diff
@@ -27,18 +27,17 @@ class test_JSONEncoder:
     def test_datetime(self):
         now = datetime.utcnow()
         now_utc = now.replace(tzinfo=pytz.utc)
-        stripped = datetime(*now.timetuple()[:3])
         serialized = loads(dumps({
             'datetime': now,
             'tz': now_utc,
             'date': now.date(),
             'time': now.time()},
         ))
         assert serialized == {
-            'datetime': now.isoformat(),
-            'tz': '{}Z'.format(now_utc.isoformat().split('+', 1)[0]),
+            'datetime': now,
+            'tz': now_utc,
             'time': now.time().isoformat(),
-            'date': stripped.isoformat(),
+            'date':  datetime(now.year, now.month, now.day, 0, 0, 0, 0),
         }
```

Comment:
Datetime sensitive **unit tests** must use [freezegun](https://pypi.org/project/pytest-freezegun/) to ensure stability of the test.

Reply:
The other alternative is to ask hypothesis to give you a datetime without a timezone. It really doesn't have to be now.